test/com/facebook/buck/go/testdata/go_test/cgo/lib/double_cgo.go

```diff
@@ -0,0 +1,12 @@
+package lib
+
+/*
```

Comment:
why is this method commented out? Remove?

Reply:
It is a method implemented in C programming language. For more information about Go calling into code written in C, please refer to the [official cgo document](https://golang.org/cmd/cgo/).